### ISSUE TITLE
Auth zone: unbound stop resolving

### ISSUE THREAD

**Issue by ArnGa** [NLnetLabs/unbound#254]
Hello,
I'm using Unbound 1.10.1 on Debian 10.
In my configuration I have multiple zone configured like this:
auth-zone:
        name: "xxx.fr"
        zonefile: /var/lib/unbound/zones/xxx.fr
        master: A.B.C.D
        allow-notify: A.B.C.D
        for-downstream: no
        for-upstream: yes
        fallback-enabled: yes
I previously used this configuration without problems in 1.9.6.
Since I've upgraded in 1.10.1, Unbound suddenly stop resolving and answer for any query about this zone: NXDOMAIN
I have several zones but I noticed this problem always on the same zone.
If I look at the serial in the zone, it is perfectly synchronized and Unbound continues to update it as the modifications take place. But it does not respond at all on the zone concerned ...

To return to normal operation, I just do an unbound-control reload and it works immediately ... for 10 minutes, 1 hour, sometimes more but it always ends up stopping.

I have seen several reports of problems indicating approaching symptoms. Wouldn't the correction of the flaw in 1.10.1 have introduced a regression?
Regards

**Comment by ralphdolmans** [comment]
Looks like Unbound after a while caches an NXDOMAIN answer, and returns that to the client. A look at the log might give you an understanding of where the NXDOMAIN is coming from. Looking at a cache dump (`unbound-control cache_dump`) might also help.

Are you sure you want the `for-downstream` to be set to no?

**Comment by ralphdolmans** [comment]
Closing this due to inactivity. Feel free to reopen if there is a reason to believe this is a bug. Feel free to use the unbound-users mailing list (https://lists.nlnetlabs.nl/mailman/listinfo/unbound-users) for questions regarding your configuration.

**Comment by ArnGa** [comment]
Sorry for the late reply.
I need for-downstream: no / for-upstream: yes, because I use Unbound for speed up the responses for the zone we are authority.
I also need dnssec validation for this zones and I think it's the correct configuration for doing this. (or not ??)

I think there is a bug about this at least in version 1.10 but I can't reproduce it anymore.
Indeed, annoyed by the bug I was forced to no longer use the auth-zone part of Unbound, now I use stub-zones to an NSD.
Occasionally I will try to reproduce the configuration under test with version 1.11 to see if anything has been changed. For now, since I found a workaround in production, I'm happy with it.
Thanks and sorry again for the late reply.
